npm/vue/package.json

```diff
@@ -12,26 +12,26 @@
     "watch": "tsc -w"
   },
   "dependencies": {
-    "@babel/plugin-transform-modules-commonjs": "7.10.4",
     "@cypress/code-coverage": "3.8.1",
```

Comment:
I think that we are not connecting it by default anymore?

Reply:
... We are, but maybe we should not.